mixins/dashboards_out/boskos-http.json

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
~Please run `make clean`.~ (This is a top level `mixins` dir not `prow/cluster/monitoring/mixins` so we just need to delete it.) We don't want to check in the generated dashboards and alerts.

Reply:
Also why are there dashboards for boskos? Those should have been removed. It looks like they were mistakenly deployed to the cluster as well so we'll need to clean those up, but we should figure out what went wrong to allow that first.

Reply:
Bad copy/paste made these files here, there is another copy at the correct path